getting-started/support_matrix.md

```diff
@@ -1,60 +1,83 @@
+[K3S]: https://www.rancher.com/products/k3s
+[OKE]: https://www.oracle.com/cloud/cloud-native/container-engine-kubernetes/
+[UEK]: https://docs.oracle.com/en/operating-systems/uek/
+[OLS]: https://docs.oracle.com/en/operating-systems/oracle-linux/8/relnotes8.7/
+[IKS]: https://www.ibm.com/cloud/kubernetes-service
+[Tanzu]: https://tanzu.vmware.com/kubernetes-grid
+[Graviton]: https://aws.amazon.com/ec2/graviton/
+[Ampere]: https://www.oracle.com/in/cloud/compute/arm/
+[1064]: https://github.com/kubearmor/KubeArmor/issues/1064
+[1084]: https://github.com/kubearmor/KubeArmor/issues/1084
+[BPFLSM]: https://github.com/kubearmor/KubeArmor/issues/484
+[Distros]: #Supported-Linux-Distributions
+
 ## Supported Linux Distributions
 
+Following distributions are tested for VM/Bare-metal based installations:
+
 | Provider | Distro | VM / Bare-metal | Kubernetes |
 |----------|--------|---------------|------|
 | SUSE | SUSE Enterprise 15 | Full | Full |
 | Debian | [Buster](https://www.debian.org/releases/buster/) / [Bullseye](https://www.debian.org/releases/bullseye/) | Full | Full |
 | Ubuntu | 18.04 / 16.04 / 20.04 | Full | Full |
-| RedHat / CentOS | RHEL 8.4 / CentOS 8.4 | Full | Partial |
-| RedHat | RHEL 9 / RHEL >= 8.5 / CentOS 8 Steam | Full | Full |
+| RedHat / CentOS | RHEL / CentOS <= 8.4 | Full | Partial |
+| RedHat / CentOS | RHEL / CentOS >= 8.5 | Full | Full |
 | Fedora | Fedora 34 / 35 | Full | Full |
 | Rocky Linux | Rocky Linux >= 8.5 | Full | Full |
+| AWS | Amazon Linux 2022 | Full | Full |
+| RaspberryPi (ARM) | Debian | Full | Full |
 
-> **Note**  
+> **Note**
 > Full: Supports both enforcement and observability  
-Partial: Supports only observability
-
-### When will EKS with Amazon Linux 2 be supported?
-
-Amazon Linux 2 currently is shipped with SELinux as the LSM (Linux Security Module). KubeArmor supports SELinux only for host-based policy enforcement. On Amazon Linux 2, Kubearmor currently supports observability/policy audits using ebpf based engine.
-
-The latest versions of Amazon Linux 2 ship with a new LSM type called BPF-LSM and Kubearmor [intends](https://github.com/kubearmor/KubeArmor/issues/484) to support it soon).
+> Partial: Supports only observability
 
 ### Platform I am interested is not listed here! What can I do?
 
 Please approach the Kubearmor community on [slack](https://github.com/kubearmor/kubearmor#slack) or [raise](https://github.com/kubearmor/KubeArmor/issues/new/choose) a GitHub issue to express interest in adding the support.
 
-It would be very much appreciated if you can test kubearmor on a platform not listed above and if you have access to. Once tested you can update this document and raise a PR, if possible.
-
-### What local K8s platforms are supported?
-
-[Minikube](../contribution/minikube), [K3s](../deployments/k3s) and [Microk8s](../contribution/microk8s) platforms are currently supported.
-
-### Why KubeArmor does not work on kind
-
-KubeArmor does not support Kubernetes in Docker.
```

Comment:
Should we mention about supporting only observability rn?

Reply:
Do not want to project local environments that we do not support fully?